src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -20,18 +33,88 @@ import {CDK_MENU, Menu} from './menu-interface';
   selector: '[cdkMenuBar]',
   exportAs: 'cdkMenuBar',
   host: {
+    '(keydown)': '_handleKeyEvent($event)',
+    '(focus)': 'focusFirstItem()',
     'role': 'menubar',
+    'tabindex': '0',
     '[attr.aria-orientation]': 'orientation',
   },
   providers: [
     {provide: CdkMenuGroup, useExisting: CdkMenuBar},
     {provide: CDK_MENU, useExisting: CdkMenuBar},
   ],
 })
-export class CdkMenuBar extends CdkMenuGroup implements Menu {
+export class CdkMenuBar extends CdkMenuGroup implements Menu, OnDestroy, AfterContentInit {
   /**
    * Sets the aria-orientation attribute and determines where menus will be opened.
    * Does not affect styling/layout.
    */
   @Input('cdkMenuBarOrientation') orientation: 'horizontal' | 'vertical' = 'horizontal';
+
+  /** Handles keyboard events for the MenuBar. */
+  private _menuKeyManager?: MenuKeyManager;
```

Comment:
```suggestion
  private _keyManager?: MenuKeyManager;
```
I would just use `_keyManager` since the "menu" part is implied